### ISSUE TITLE
[Skype] HueBot sample: Skype bot cannot connect to the service using skype

### ISSUE THREAD

**Issue by RomanMolchanov2017** [Microsoft/BotBuilder#2774]
I followed instructions for registering a bot (https://docs.microsoft.com/en-us/bot-framework/portal-register-bot) and deploying a real-time audio-video bot (https://docs.microsoft.com/en-us/bot-framework/dotnet/bot-builder-dotnet-real-time-deploy-visual-studio) to run the HueBot sample (https://github.com/Microsoft/BotBuilder-RealTimeMediaCalling/tree/master/Samples/HueBot).
But after my bot was added to my skype contacts I cannot make a video call to my bot. I always get a response like here: "You can't talk to this bot just yet. But we are working on it." And at the bot framework dashboard, I have the issues: "There was an error completing this request due to an internal channel failure."
I connected the Visual Studio 2017 debugger to the published job worker and saw that the controller didn't receive any requests.
Bot configuration:
Messaging endpoint: https://dataartskypebots.cloudapp.net
App ID: bf0f99b0-1836-4548-b1c6-6540ee25d191
Enable messaging: Yes;
Enable autoplay of media cards: Yes;
Enable calling: Yes;
Real Time Media: Yes;
Video (Audio alway enabled): Yes;
Screen sharing: Yes;
Webhook (for calling): https://dataartskypebots.cloudapp.net/api/calling/call
Enable adding to group: Yes;

Could you help me what is incorrect in the bot configuration or maybe you have any other recommendations?

**Comment by nwhitmont** [comment]
@RomanMolchanov2017 Are you using .NET or Node SDK for BotBuilder?

On initial inspection, your messaging endpoint should be: `https://dataartskypebots.cloudapp.net/api/messages`

**Comment by nwhitmont** [comment]
@RomanMolchanov2017 are you still having issues after updating your messaging endpoint to `https://dataartskypebots.cloudapp.net/`?

**Comment by RomanMolchanov2017** [comment]
HueBot sample, I pointed to in my issue, didn't have code to process text messages at all at the moment I opened this issue. So this message endpoint doesn't have sense. This bot only can receive calls. I received the clarification from the Microsoft/BotBuilder-RealTimeMediaCalling team: the bot can't use a self-signed certificate with the private key (.pfx) but needs a certificate from trusted CA.
Anyway, thank you for your assistance.
